src/Components/Web.JS/src/Platform/Logging/Loggers.ts

```diff
@@ -12,28 +12,29 @@ export class NullLogger implements Logger {
 }
 
 export class ConsoleLogger implements Logger {
-  private readonly minimumLogLevel: LogLevel;
+  private readonly minLevel: LogLevel;
```

Comment:
Why does this still show up in the terse output?

Reply:
TypeScript private fields are just convention; they are still public on exported classes, so it doesn't drop the names.

![image](https://user-images.githubusercontent.com/1142958/108781280-8941cb00-7561-11eb-963a-4c57f389ab98.png)

It will mangle JavaScript private fields (ones which start with `#`); however if you use them in TypeScript it compiles them to references to a `WeakMap` rather than converting them to JavaScript private fields; which is unideal.

Terser does have a "Mangle properties" option; but that basically wrecks the output so it doesn't run if you don't play with the settings, was considering doing it in a follow-up when I workout what works.

Reply:
Looks like might be able to solve it with
```js
mangle: {
    properties: {
        regex: /^_/
    }
},
```
and then prefixing every private field/method with `_`